services/fromage/README.md

```diff
@@ -0,0 +1,16 @@
+# FROMAGe Service
+**FROMAGe** is a service that is used to get an image and respond accordingly to the user's questions. FROMAGe is based on grounding pretrained language models to the visual domain ([Grounding Language Models to Images for Multimodal Inputs and Outputs](https://arxiv.org/abs/2301.13823)). 
```

Comment:
**FROMAGe** is a service that is used to process an input image and respond to the user's questions accordingly. It is based on the [FROMAGe](https://github.com/kohjingyu/fromage/tree/main) model from [Grounding Language Models to Images for Multimodal Inputs and Outputs](https://arxiv.org/abs/2301.13823).